release-please-config.json

```diff
@@ -1,16 +1,24 @@
 {
+  "monorepo-tags": true,
+  "tag-separator": "/",
   "bootstrap-sha": "c4e030ed651ab800aa56a555471da5913eb95259",
   "packages": {
-    ".": {
+    "cmd/flagd": {
```

Comment:
How will shared resources work? Do you know if dependencies can be defined?

Reply:
This only works if we are able to leverage a plugin that parses the dependency tree. There exists one in node but not Go, I've written more about this [here](https://github.com/open-feature/flagd/issues/435#issuecomment-1448576876).

Reply:
Yeah, sorry about that. I just noticed you include a nice write up already in the issue.